src/TestRunnerCde/appsettings.json

```diff
@@ -19,7 +19,7 @@
   },
   "Cde": {
     "UseRandomScopeId": false,
-    "ScopeId": "",
+    "ScopeId": "321123",
```

Comment:
We should not set a `ScopeId` for the Cde TestRunner. If the entry is not set, the CDE infrastructure implementation generates a random scope-id.